src/MCPClient/lib/clientScripts/archivematicaCreateMETS2.py

```diff
@@ -145,13 +143,19 @@ def getDublinCore(unit, id):
     except DublinCore.DoesNotExist:
         return
 
-    ret = etree.Element(ns.dctermsBNS + "dublincore", nsmap={"dc": ns.dctermsNS})
+    ret = etree.Element(ns.dctermsBNS + "dublincore", nsmap={"dcterms": ns.dctermsNS, 'dc': ns.dcNS})
     ret.set(ns.xsiBNS + "schemaLocation", ns.dctermsNS + " http://dublincore.org/schemas/xmls/qdc/2008/02/11/dcterms.xsd")
 
-    for term in field_list:
-        txt = getattr(dc, term)
+    for dbname, term in db_field_mapping.iteritems():
+        txt = getattr(dc, dbname)
+        elem_ns = ''
+        # See http://dublincore.org/documents/2012/06/14/dcmi-terms/?v=elements for which elements are which namespace
+        if term in ('contributor', 'coverage', 'creator', 'date', 'description', 'format', 'identifier', 'language', 'publisher', 'relation', 'rights', 'source', 'subject', 'title', 'type'):
+            elem_ns = ns.dcBNS
+        elif term in ('isPartOf'):
```

Comment:
Nitpick, but just check for string equality? Otherwise, this needs an extra comma (it's being evaluated as `'isPartOf`, not as a one-length tuple)